projects/make-a-vis/src/themes/shared/base.scss

```diff
@@ -205,3 +204,202 @@
 .visualization-container .tooltip {
   font-family: $tooltip-font !important;
 }
+
+/* colors from table.component.scss */
+.mav-datatable-container {
+  .datatable-wrapper {
+    border-color: #e0e0e0;
+    .datatable-container {
+      .card-header {
+        .label {
+          color: #212121;
```

Comment:
colors should be pulled from the variables ($gray-accent-1, etc.) unless there is no corresponding one, in which case we need to see if that color is the one Leonard actually intended.